src/faq.md

```diff
@@ -265,7 +320,7 @@ For command-line flags that are either **mutually exclusive** or **dependent**,
 ```
 ## Setting Mutually Exclusive Parameters
```

Comment:
Again - this is not how any of the other FAQ titles are currently formatting.  I personally find a longer question harder to parse and doesn't give me any more information.  If you want to add additional content sure, but I don't think we need to make them questions - it is sort of implied by the FAQ title.

Reply:
We could actually think about removing the FAQ section entirely and trying to weave these into the whole User Guide.  A current school of thought is that FAQs aren't the best idea because they break the narrative and flow and you don't get the information in context.  Something to think about for the future.

Reply:
Hmm, `main` contains "How do I set mutually exclusive parameters?". So that's a question, but not 100% the suggested text. I'll create an issue for this just in case: https://github.com/common-workflow-language/user_guide/issues/393